tools/doc/type-parser.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+const nodeDocUrl = '';
+const jsDocUrl = 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/' +
+  'Reference/Global_Objects/';
+const jsGlobalTypes = [
+  'Error', 'Number', 'Object', 'Boolean', 'String', 'Function', 'Symbol',
+  'Array', 'Uint8Array', 'Uint16Array', 'Uint32Array', 'Int8Array',
+  'Int16Array', 'Int32Array', 'Uint8ClampedArray', 'Float32Array',
+  'Float64Array', 'Date', 'RegExp', 'ArrayBuffer', 'DataView', 'Promise',
+  'Null'
```

Comment:
I also am not sure anything actually use [`Uint8ClampedArray`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8ClampedArray)? A search didn't bring it up in the compiled docs.